changes/8891.general.rst

```diff
@@ -0,0 +1 @@
+Have pytest try to colorize its output.
```

Comment:
I don't think this needs a change log entry since there's no impact on users.  We can remove this note.